src/Drivers/AbstractSMS.php

```diff
@@ -6,6 +6,9 @@
 
 abstract class AbstractSMS
 {
+
+    protected $debug;
```

Comment:
Move this to the driver or drop it completely.  I'm not sure exactly what debug mode does with this provider but it does not apply to the other drivers.